packages/terra-popup/src/PopupArrow.module.scss

```
@@ -3,82 +3,82 @@
   [dir=rtl] {
     // These are relative to screen coordinates, so no bi-di concerns
     .popup-arrow {
-      border: var(--terra-popup-arrow-size, 11px) solid transparent;
+      border: var(--terra-popup-arrow-border, 11px) solid transparent;
       display: none;
       height: 0;
       left: 50%;
       position: absolute;
       top: 50%;
       width: 0;
-      z-index: var(--terra-popup-arrow-z-index, 7002);
+      z-index: 7002;
 
       &::after {
-        border: var(--terra-popup-arrow-inner-size, 10px) solid transparent;
+        border: var(--terra-popup-arrow-after-border, 10px) solid transparent;
```

Let's update this to 
```suggestion
      border-color: transparent;
      border-style: solid;
      border-width: var(--terra-popup-arrow-after-border-width, 11px); 
```